journals/apps/journals/models.py

```diff
@@ -598,19 +604,29 @@ def update_related_objects(self, deactivate=False):  # pylint: disable=missing-d
             self.journal.name = self.title
             self.journal.save()
 
-        update_service(
-            self.site.siteconfiguration.discovery_journal_api_client,
-            self.journal.uuid,
-            discovery_data,
-            "discovery"
-        )
+        try:
```

Comment:
@bill-filler we should add this error handling inside `update_service` definition [here](https://github.com/edx/journals/blob/master/journals/apps/journals/api_utils.py#L22)

Reply:
@ziafazal 
fixed